tasks/hivemind/agent.py

```
@@ -156,35 +154,21 @@ def detect_question_type(self) -> str:
-        agent_executor = AgentExecutor(
-            agent=agent,
-            tools=tools,
-            verbose=True,
-            return_intermediate_steps=False,
-            max_iterations=3,
-        )
+        try:
+            answer = asyncio.run(query_data_sources.query(self.state.user_query))
+            if answer is None:
+                answer = "NONE"
+        except Exception as e:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)